test/TypeTraitsTest.cpp

```diff
@@ -17,15 +20,74 @@ TEST(TypeTraits, IsSimilar) {
   ASSERT_TRUE(true);
 }
 
-TEST(TypeTraits, IsContained) {
+/*
+Call the given lambda with explicit types: `std::decay_t<T>`,`const
+std::decay_t<T>`,`std::decay_t<T>&`,`const std::decay_t<T>&`.
+*/
+template <typename T>
+constexpr void callLambdaWithAllVariationsOfType(const auto& lambda) {
+  using decayedT = std::decay_t<T>;
+  lambda.template operator()<decayedT>();
+  lambda.template operator()<const decayedT>();
+  lambda.template operator()<decayedT&>();
+  lambda.template operator()<const decayedT&>();
```

Comment:
If you really want you can also add `&&` and `const &&` :)
(And also volatile, but that is really not relevant for anything we do)

Reply:
Done, see [8182471](https://github.com/ad-freiburg/qlever/pull/1214/commits/8182471141cfe0609dac936feaf67b5b20245a67).